### PR TITLE
Update youfoundalpha.json

### PR DESCRIPTION
**Pull request by YouFoundAlpha**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/youfoundalpha.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "Mastodon": "@YouFoundAlpha@mastodon.social"
   },
   "record": {
-   "A": ["89.106.200.1"],
+   "CNAME": "https://ap.yfadev.pw",
     "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
     "TXT": "v=spf1 include:spf.improvmx.com ~all"
```
Comment (CuteDog5695) [inline domains/youfoundalpha.json:13]: you cant use CNAME with any other record (also you cant have https:// on CNAME)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by YouFoundAlpha — COMMENTED** [review]
Fixed

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Comment by YouFoundAlpha** [comment]
Thank you

**Pull request merged**